source/Government.h

```diff
@@ -152,6 +154,10 @@ class Government {
 	double crewAttack = 1.;
 	double crewDefense = 2.;
 	bool provokedOnScan = false;
+	// If a government appears in this set, and the reputation with this government is affected by actions,
+	// and events performed against that government,
+	// use the penalties that government applies for the action instead of this governments own penalties.
```

Comment:
```suggestion
	// If a government appears in this set, and the reputation with this government is affected by actions,
	// and events performed against that government, use the penalties that government applies for the
	// action instead of this governments own penalties.
```